graphql/tests/image/distros/data.json

```diff
@@ -0,0 +1,27 @@
+{
+    "image": [
+      {
+        "_id": "ubuntu1604"
+      }
+    ], 
+    "distro": [
+        {
+          "name": "ubuntu1604-large",
+          "image_id": "ubuntu1604"
+        },
+        {
+            "name": "ubuntu1604-small",
+            "image_id": "ubuntu1604"
```

Comment:
nit about formatting; the spacing is off between these two items. You can probably download a plugin that formats the json in a neat way!

Reply:
Got a json plugin~